pre-commit

```diff
@@ -19,6 +19,14 @@ if [ ! -e "$DEV_LIB_PATH/check-diff.sh" ]; then
 	exit 1
 fi
 
+DEV_LIB_COMMIT_MESSAGE_FILE=".dev-lib-commit-message.$PPID"
+DEV_LIB_COMMIT_MESSAGE=''
+function add_commit_message {
+	label=$1
+	value=$2
+	DEV_LIB_COMMIT_MESSAGE+="\n$label: $value"
```

Comment:
The files are super tiny, so I'm not so concerned about it, since this is going to be used on dev environments not servers. Also, there are a lot of other files being written to `/tmp` when running the `pre-commit` hook and Travis CI builds (see usages of `TEMP_DIRECTORY` in https://github.com/xwp/wp-dev-lib/blob/master/check-diff.sh).